expression/builtin_other_vec.go

```diff
@@ -79,11 +79,32 @@ func (b *builtinValuesJSONSig) vecEvalJSON(input *chunk.Chunk, result *chunk.Col
 }
 
 func (b *builtinBitCountSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinBitCountSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		if err != nil && types.ErrOverflow.Equal(err) {
+			return err
```

Comment:
This is different from the non-vectorized implementation. We need to set the result to 64.

Reply:
Should we set all the values of this column in this chunk to 64? It seems still behave differently from the previous implementation.

Reply:
I knew it, but it's hard to be solved now.
What's your opinion? @qw4990 @SunRunAway

Reply:
One option is to use `b.args[0].EvalInt(b.ctx, row)` to evaluate the argument.